machine_learning/logistic_regression.py

```diff
@@ -55,6 +62,12 @@ def logistic_reg(
         J = cost_function(h, y)
 
         iterations += 1  # update iterations
+        
+         if step % 10000 == 0:
```

Comment:
The Travis CI tests fail because there is one space too many here.  Python is picky about indentation.